src/app/App.tsx

```diff
@@ -0,0 +1,12 @@
+import { BrowserRouter } from 'react-router-dom';
+import AppRoutes from './routes';
+
+const App = () => {
+  return (
+    <BrowserRouter>
```

Comment:
react creaters recommend use react strictmode, it may prevent bugs https://react.dev/reference/react/StrictMode